gapic/batch/staging.yaml

```diff
@@ -0,0 +1,5 @@
+common:
```

Comment:
This file is no longer staging-specific. Maybe one of these options for the filename:

* gapic/core/batch.yaml
* gapic/batch/common.yaml